build.sbt

```diff
@@ -1,11 +1,19 @@
+lazy val rcVersion = "3.0.0-RC1"
+lazy val crossVersions = Seq(Option(rcVersion), dottyLatestNightlyBuild()).flatten
 lazy val root = project
   .in(file("."))
   .settings(
     name := "scala3-example-project",
     description := "Example sbt project that compiles using Scala 3",
     version := "0.1.0",
-
-    scalaVersion := "3.0.0-RC1",
-
+    scalaVersion := rcVersion,
+    crossScalaVersions := crossVersions,
     useScala3doc := true,
+    skip in publish := true
   )
+
+ThisBuild / githubWorkflowJavaVersions := Seq("adopt@1.8", "adopt@11")
```

Comment:
perhaps adopt@16 too?

Reply:
Yeah that's a good idea. I can add that in tonight!